src/components/account/details/Button.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react'
+
+import '../../../styles/account/details/button.css'
+
+export default class Button extends Component {
+  render() {
+    return (
+      <div className="button">
+        <span>{ this.props.name }</span>
```

Comment:
When we're creating components that just wrap some other stuff (text, or components), we can make use of `this.props.children`, which allows us to use the components in a slightly more intuitive way, without adding additional props to remember to pass down:
```jsx
render() {
  return (
    <div className="button">
      <span>{ this.props.children }</span>
    </div>
  )
}
```
Now, we can do:
```jsx
<Button>Text</Button>
```